todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from todo_app.view_model import ViewModel
+from datetime import datetime, timedelta
+import os 
+from todo_app.todo_item import TodoItem
+
+
+@pytest.fixture
+def view_model():
+    _DEFAULT_CARDS = [
+        {
+            "idShort": 1,
+            "idList": os.getenv("to_do_list"),
```

Comment:
These tests depend on these environment variables being set, the list ids have to be different or the logic in todo_item.py will not work. If you run e.g. `poetry run pytest todo_app/tests/test_view_model.py` they won't be.

You could load `.env.test` like you do for the integration tests, or manually set them at the start of this function.

Reply:
I have now loaded .env.test in my test_view_model. Please could you review. Thanks